libkbfs/folder_block_ops.go

```diff
@@ -694,43 +675,43 @@ func (fbo *folderBlockOps) GetDirtyEntry(
 	return fbo.getDirtyEntryLocked(ctx, lState, md, file)
 }
 
+func (fbo *folderBlockOps) getOrCreateDirtyFileLocked(lState *lockState,
```

Comment:
maybe you're already planning this, but it might be nice to pass the full path to this function and keep track of it in `dirtyFile`, so that any errors can print a path instead of just a block pointer, and to make it possible to expose to a special file / UI a list of dirty files and their status. (Not for this PR of course)